ide/parsing.indexing/nbproject/project.properties

```diff
@@ -14,7 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-javac.source=1.8
+javac.release=17
 javac.compilerargs=-Xlint -Xlint:-serial
 spec.version.base=9.39.0
```

Comment:
```suggestion
spec.version.base=9.40.0
```

Reply:
All spec versions should have just been incremented. Changing them during delivery phase is a pain.